.github/dependabot.yml

```diff
@@ -1,11 +1,26 @@
 version: 2
 updates:
   - package-ecosystem: "cargo"
-    directory: "/"
+    directories:
+      - "/"
+      - "bench-build"
+      - "examples/*"
+      - "fuzzing"
+      - "fuzzing/fuzz"
     schedule:
       interval: "daily"
 
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "daily"
+
+  - package-ecosystem: "gitsubmodule"
```

Comment:
Can you explain what's this and the `pip` one below please?

Reply:
gitsubmodule updates .gitmodules (you included fuzzing-corpus), and pip updates pyproject.toml (you declared 2 dependencies).
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#package-ecosystem-